test/e2e/page-objects/pages/transaction-list-page.ts

```diff
@@ -0,0 +1,34 @@
+import { Driver } from '../../webdriver/driver';
+
+export class TransactionListPage {
+  private driver: Driver;
+
+  constructor(driver: Driver) {
+    this.driver = driver;
+  }
+
+  async goToActivityTab() {
+    await this.driver.clickElement(
+      '[data-testid="account-overview__activity-tab"]',
+    );
```

Comment:
We already have a class for the Home page which goes to activity list:

https://github.com/MetaMask/metamask-extension/blob/main/test/e2e/page-objects/pages/home/homepage.ts#L100